include/quiche.h

```diff
@@ -270,6 +270,16 @@ bool quiche_conn_is_in_early_data(quiche_conn *conn);
 // Returns true if the connection is closed.
 bool quiche_conn_is_closed(quiche_conn *conn);
 
+// Initializes the stream's application data.
```

Comment:
The internal docs state this can only be done once, is that worth repeating in the `.h`?

Reply:
Added.